File: Basics/Dart/Q9.dart
import 'dart:io';

void main() {
  print("Enter a number: ");
  int num1 = int.parse(stdin.readLineSync()!);
  print("Enter a another number: ");
  int num2 = int.parse(stdin.readLineSync()!);
  print("Enter an operator (+, -, *, /): ");
  String operators = stdin.readLineSync()!;

  switch (operators) {
    case "+":
      print("Addition: ${num1 + num2}");
      break;
    case "-":
      print("Subtraction: ${num1 - num2}");
      break;
    case "*":
      print("Multiplication: ${num1 * num2}");
      break;
    case "/":
      print("Division: ${num1 / num2}");
      break;
    default:
      print("Invalid operator");
  }
}

File: Basics/JavaScript/Q9.js
let input = prompt("Enter the number: ")
let input2 = prompt("Enter the another number: ")
let input3 = prompt("Enter the operator: ")

let num1 = parseInt(input), num2 = parseInt(input2), operator = input3;


switch (operator) {
     case "+":
          console.log(`Addition: ${num1 + num2}`);          
          break;
     case "-":
          console.log(`Subtract: ${num1 - num2}`);          
          break;
     case "*":
          console.log(`Multiply: ${num1 * num2}`);          
          break;
     case "/":
          console.log(`Divide: ${num1 / num2}`);          
          break;

     default:
          console.log("Invalid operator");
          break;
}